### PR TITLE
docs(changelog): update changelog for tag v0.5.11

### PR DESCRIPTION
**Pull request by h-suzuki-isp**
## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
 
 ## CARET
 
+### v0.5.11 <small>\_ Sep 30, 2024</small> {id = "0.5.11"}
+
+- **Update**: Fixed a problem where only one legend is displayed even if the maximum number of legends is exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))
+
+- **Update**: Fixed mypy1.11 error untyped-overrides. ([caret_analyze #527](https://github.com/tier4/caret_analyze/pull/527))
+
+- **Update**: Fixed stackedbar when time reversal occurs. ([caret_analyze #517](https://github.com/tier4/caret_analyze/pull/517))
```
Comment (ymski) [inline docs/changelog.md:11]: Please use stacked bar graph or `StackedBar`.

File: docs/changelog.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+
+- **Update**: Fixed type_check_decorator. ([caret_analyze #513](https://github.com/tier4/caret_analyze/pull/513))
+
+- **Update**: Refactoring of AssignContextReader class. ([caret_analyze #532](https://github.com/tier4/caret_analyze/pull/532))
+
+- **Update**: refactoring for the case of merging with source timestamp. ([caret_analyze #526](https://github.com/tier4/caret_analyze/pull/526))
```
Comment (ymski) [inline docs/changelog.md:17]: The changes are not clear. Can you be more specific?

File: docs/changelog.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
 
 ## CARET
 
+### v0.5.11 <small>\_ Sep 30, 2024</small> {id = "0.5.11"}
+
+- **Update**: Fixed a problem where only one legend is displayed even if the maximum number of legends is exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))
```
Comment (ymski) [inline docs/changelog.md:7]: The content seems a bit inaccurate. How about the following text?

```suggestion
- **Update**: Fixed a probelm where a warning was incorrectly displayed when the maximum number of legends was not exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))
```

**Comment by ymski** [comment]
@h-suzuki-isp 
Could you please change the label to match the PR?

File: docs/changelog.md (hunk 2 above)
Comment (h-suzuki-isp) [reply docs/changelog.md]: Fixed in https://github.com/tier4/caret_doc/pull/356/commits/ad0751c8e11b35fc512449ec0dfb7c6656a329eb.

File: docs/changelog.md (hunk 1 above)
Comment (h-suzuki-isp) [reply docs/changelog.md]: Fixed in https://github.com/tier4/caret_doc/pull/356/commits/ad0751c8e11b35fc512449ec0dfb7c6656a329eb.

**Review by ymski — APPROVED** [review]
LGTM

**Pull request merged**